### PR TITLE
PR for sealer run support podcidr #169

### PR DESCRIPTION
**Pull request by berlinsaint**
PR for sealer run support podcidr 

### Does this pull request fix one issue?

None

### CODE REVIEW

**Comment by berlinsaint** [comment]
@cuisongliu @fanux

**Comment by cuisongliu** [comment]
ci is error . maybe linter error , you can see [CONTRIBUTING.md](https://github.com/alibaba/sealer/blob/main/CONTRIBUTING.md). and  annotation fix to English.

File: apply/run.go [hunk 1]
```diff
@@ -43,6 +45,14 @@ func IsIPList(args string) bool {
 	return true
 }
 
+func IsCidrString(arg string) bool {
+	_, err := utils.ParseCIDR(arg)
+	if err != nil {
```
Comment (fanux) [inline apply/run.go]: just return err == nil

Comment (berlinsaint) [reply apply/run.go]: done

File: utils/cidr.go [hunk 2]
```diff
@@ -0,0 +1,74 @@
+package utils
+
+import (
+	"encoding/hex"
+	"net"
+)
+
+/*
+	https://en.wikipedia.org/wiki/Classless_Inter-Domain_Routing
+	CIDR表示法:
```
Comment (fanux) [inline utils/cidr.go:10]: using english

**Review by fanux — APPROVED** [review]

**Comment by fanux** [comment]
LGTM 
/need test

File: sealer/cmd/run.go [hunk 3]
```diff
@@ -54,4 +54,5 @@ func init() {
 	runCmd.Flags().StringVarP(&runArgs.Pk, "pk", "", cert.GetUserHomeDir()+"/.ssh/id_rsa", "set baremetal server private key")
 	runCmd.Flags().StringVarP(&runArgs.PkPassword, "pk-passwd", "", "", "set baremetal server  private key password")
 	runCmd.Flags().StringVarP(&runArgs.Interface, "interface", "i", "", "set default network interface name")
+	runCmd.Flags().StringVarP(&runArgs.PodCidr, "podcidr", "", "", "set default pod CIDR network. example '192.168.1.0/24'")
```
[the remark this answers is not in the document]
Comment (fanux) [reply sealer/cmd/run.go:57]: No, default values should be image default Clusterfile, defined by image maker

**Pull request merged**